source/triggers/database-triggers.txt

```diff
@@ -591,22 +594,22 @@ multiple executions of a Trigger Function occur in event order.
 
 Disable Collection-Level Preimages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-          
-Document preimages use extra information stored in the oplog.
-The extra data may degrade trigger performance.
-
-Once you've enabled document preimages for any trigger on a
-given collection, that collection will include preimage data in
-the oplog and other triggers on the collection can use preimages
-with no additonal overhead.
-
-You can disable document preimages on a per-trigger basis to
-exclude the preimage from change events. Regardless of your
-trigger-level settings, a collection's oplog entries will
-continue to include preimage data unless you explicitly disable
-preimages for the collection.
 
-:ref:`Learn how to view and disable collection-level preimages. <preimages>`
+Document preimages require your cluster to record additional data about
+each operation on a collection. The additional storage space and compute
+overhead may degrade trigger performance depending on your cluster
+configuration.
+
+Once enabled for any trigger on a collection, your cluster stores
+preimages for every operation on the collection. The collection-level
+setting is different than the preimages setting for individual triggers.
+This means that the collection records preimages even if you later
+delete or exclude preimages from all triggers on the collection.
```

Comment:
taking the above comment https://github.com/mongodb/docs-app-services/pull/375/files#r1123699911 into account, the copy in this paragraph could prob be refined a bit, like deleting the sentence "The collection-level
setting is different than the preimages setting for individual triggers."